rekuho/src/lib/Rekuho.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts" context="module">
+  import kanagawa from "./kanagawa.js"
+  import type { Theme } from "./Theme"
+
+  const websocket = async () => {
+    try {
+      const ws = new WebSocket("ws://localhost:6969")
+      const cmd = {
+        method: "key_pressed",
+        params: {
+          key: "X",
+          modifiers: [],
+        },
+      }
+
+      await new Promise((resolve) => ws.addEventListener("open", () => resolve(null)))
+      ws.send(JSON.stringify(cmd))
+    } catch (e) {
+      console.log(e)
+    }
+  }
+
+  export let theme: Theme = {
+    font: {
+      family: "monospace",
+      weight: "normal",
+      size: "20px",
+    },
+    text_offset: 10,
+    gutter: {
+      background: kanagawa.sumiInk0,
+      width: 50,
+    },
+
+    font_color: kanagawa.fujiWhite,
+    primary_cursor_color: kanagawa.fujiWhite,
+    editor_background: kanagawa.sumiInk1,
+  }
+</script>
+
+<script lang="ts">
+  import Portion from "./Portion.svelte"
+
+  const _ = websocket()
```

Comment:
??

Reply:
Don't worry about it

Reply:
I'm running the websocket, haven't really spent much time. We spoke of using websockets solely for the purpose of prototyping and debugging. Now the idea is to keep it around for development purposes, potentially even the main RPC for the webui.

Reply:
Fixed.